app/assets/javascripts/templates/price_breakdown.html.haml

```diff
@@ -1,4 +1,4 @@
-.joyride-tip-guide.price_breakdown{ng: {class: "{ in: tt_isOpen, fade: tt_animation }", show: "tt_isOpen"}}
+.joyride-tip-guide.price_breakdown{id: ".price_breakdown", ng: {class: "{ in: tt_isOpen, fade: tt_animation }", show: "tt_isOpen"}}
```

Comment:
Sorry, I wasn't clear enough. 
I don't think you need to add anything on this file since that there is already a class that specify the purpose: `.price_breakdown` (juste before the `{`)
I'd vote to not modify this file at all ;)